evap/evaluation/tests/test_tools.py

```diff
@@ -62,20 +62,26 @@ def test_log_exceptions_decorator(self, mock_logger, __):
 
 
 class TestHelperMethods(WebTest):
-    def test_is_many_prefetched(self):
-        evaluation = baker.make(Evaluation)
+    def test_is_prefetched(self):
+        evaluation = baker.make(Evaluation, voters=[baker.make(UserProfile)])
         baker.make(Contribution, evaluation=evaluation)
 
-        self.assertFalse(is_many_prefetched(evaluation, "contributions"))
+        def test_logic(cls, pk: int, field: str):
```

Comment:
I was playing around here a bit, I think the best annotation would be `Type[Model]`, but mypy also accepts just `Type`, because that is `Type[Any]`, but not `Type[object]`. Anyways, I suggest `Type[Model]` :^)